pegasus/migrations/096_drop_unique_constraint_from_forms_source_id_index.rb

```diff
@@ -0,0 +1,22 @@
+Sequel.migration do
+  up do
+    # For some reason this unique index only exists on production, but not staging or test.
+    # Conditionally drop it if it exists.
+    if self.indexes(:forms)[:unique_forms_source_id]
+      alter_table(:forms) do
+        drop_index :source_id, name: :unique_forms_source_id
+      end
+    end
+
+    # Replace with a unique index on source_id and kind (rather than just source_id).
+    alter_table(:forms) do
+      add_index [:source_id, :kind], unique: true
+    end
+  end
```

Comment:
Do we have any idea how long this migration will take in production? Do we have an understanding of what impact there will be on production, given that it will take a long time?

Does it make sense to break this migration into two migrations (merged in separate DTPs) so as to have two shorter migrations rather than one long migration?